atlp-rwanda/cabal-phantom-frontend
# #7rbb9Mkc Setup Redux

### **What about this PR?**
This PR has setup of redux

### **Description of Task to be completed**
It should have the following endpoint:
- Setup Redux

### **How should be tested?**
- Check the counter through [here](https://cabal-fronte-ch-setup-r-i93ydx.herokuapp.com/testRedux)
- Also, you can install [Redux DevTools Extension](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd) and check action and state
 
### **Relevant pivots tracker**
[#7rbb9Mkc](https://trello.com/c/7rbb9Mkc/26-setup-redux)
